apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -274,14 +274,18 @@ export default function CourseMap() {
      * Every two markers grouped as [start, destination] tuples for the routes.
      */
     const startDestPairs = useMemo(() => {
-        return markersToDisplay.reduce((acc, cur, index) => {
-            acc.push([cur]);
-            if (index > 0) {
-                acc[index - 1].push(cur);
-            }
-            return acc;
-        }, [] as (typeof markersToDisplay)[]);
-    }, [markersToDisplay]);
+        const allEvents = [...markersToDisplay, ...customEventMarkersToDisplay];
```

Comment:
`customEventsMarkersToDisplay` doesn't update when new events are made/updated because there's no listener and no code in useEffect that would update it.

Reply:
Makes sense! But will it be better if I add listeners in useEffect so changes are able to be reflected?

Reply:
Oh yes, that's the proper way to do it; sorry for not communicating my feedback more clearly!

Reply:
Looks like a duplicate to https://github.com/icssc/AntAlmanac/pull/855 0.0

Reply:
Sure, why not.